src/main/java/org/broadinstitute/hellbender/utils/codecs/gencode/GencodeGtfFeature.java

```diff
@@ -108,23 +109,24 @@ protected GencodeGtfFeature(final String[] gtfFields) {
         // But we need to match up the field names to the fields themselves:
         for ( final String extraField : extraFields ) {
 
-            final String[] fieldParts = extraField.trim().split(EXTRA_FIELD_KEY_VALUE_SPLITTER);
-
-            if ( fieldParts.length == 1 ){
-                if ( fieldParts[EXTRA_FIELD_KEY_INDEX].isEmpty() ){
-                    continue;
-                }
-                else {
-                    throw new UserException.MalformedFile("Extraneous optional field data - not in a key/value pair: " + extraField);
-                }
+            final String trimmedExtraField = extraField.trim();
+            if (trimmedExtraField.isEmpty()) {
+                continue;
             }
 
-            // Each optional field is in a key/value pair:
-            final String fieldName = fieldParts[EXTRA_FIELD_KEY_INDEX].trim();
-
-            // The value of the field may be between two quotes.
-            // We remove them here.
-            final String fieldValue = fieldParts[EXTRA_FIELD_VALUE_INDEX].trim().replaceAll("\"", "");
+            final int splitPoint = trimmedExtraField.indexOf(EXTRA_FIELD_KEY_VALUE_SPLITTER);
+            final String fieldName;
+            final String fieldValue;
+            if(splitPoint == -1 ) {
+                throw new UserException.MalformedFile("Extraneous optional field data - not in a key/value pair: " + extraField);
+            } else {
+                fieldName = trimmedExtraField.substring(0,splitPoint).trim();
+
+                // The value of the field may be between two quotes.
+                // We remove them here.
+                final String rawFieldValue = trimmedExtraField.substring(splitPoint + 1, trimmedExtraField.length());
```

Comment:
If there's more than 1 occurrence of the delimiter, the extra tokens will now all be concatenated together in `rawFieldValue`, whereas before they would be ignored. What's the desired behavior here? Should multiple delimiters be an error condition?

Reply:
@jonn-smith Can you answer the case of what should happen if there are multiple delimiters here? Previous behavior was that everything after the second element in the split array would be ignored.  It seems like maybe it should be an error condition to have >2 elements though?

Reply:
I'm assuming that's an error case and added a check for now

Reply:
I think that's fine for now.  I don't remember if I ran into an issue that required that I ignore the second delimiter on.